### PR TITLE
Project description

### PR DESCRIPTION
**Pull request by vincentott**
This is my first pull request.
It adds the R package structure, the documentation, as well as a basic function 'play()' which starts the shiny app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,16 @@
-# cellular-automata
-Conway´s and other games of life
+# Shiny Like-Living Cells
+An R Shiny App for Conway´s and other Games of Life
```
Comment (vincentott) [inline README.md]: so now i can leave a comment here, huh?

**Comment by vandenman** [comment]
Please don't merge the pull request yourself next time but wait until class!

**Comment by vincentott** [comment]
okay, thank you!

File: .Rhistory [hunk 2]
```diff
@@ -0,0 +1,62 @@
+hello()
```
Comment (mfhstdt) [inline .Rhistory:1]: I guess you can delete your Rhistory from github since it has no relevance for the package

**Review by mfhstdt — COMMENTED** [review]
Nice project idea and its cool that you already got quite far! 
The package structure is not missing any elements and its good that you worked with roxygen. 
I recommend deleting your Rhistory from the repo and having a closer look at the .Rproj.user folder. Check if it is necessary for the project (if it isn't it might make sense to delete it from the repo since it makes the pull requests a bit confusing). :)

File: .Rhistory (hunk 2 above)
Comment (vincentott) [reply .Rhistory:1]: Thank you very much! I will look into this throughout the next few days!

**Comment by MagdaMatetovici** [comment]
Nice project - the package structure looks good :)! Maybe delete also the .DS_store files.

**Pull request merged**